client/datasync-client/src/DataManager.ts

```diff
@@ -138,6 +213,18 @@ export class DataManager {
   }
 
   /**
+   * Get the current number of pending sync requests for this data set.
+   */
+  private getQueueSize = function() {
+    const self = this;
+    return new Bluebird(function(resolve) {
+      syncApi.getPending(self.datasetId, function(pending?) {
```

Comment:
Whats the `?`

Reply:
short form of `pending?:any`

Reply:
`pending: any | undefined`

Reply:
Is there a way to handle errors for this?

Reply:
Actually I was thinking about this as well and aparently this code is just fetching local data. Errors will just return empty object.

Reply:
:+1: I thought that might be the case, maybe of an error value like -1 or so.